onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -5567,8 +5544,7 @@ def aten_trace_backward(grad: TensorType, sizes: INT64) -> TensorType:
 def aten_transpose(self, dim0: int, dim1: int):
     """transpose.int(Tensor(a) self, int dim0, int dim1) -> Tensor(a)"""
 
-    # FIXME(justinchuby): onnxscript raises Unsupported expression type
-    # Script the function when this is fixed
+    # Use trace only to construct the prem attribute in Transpose
```

Comment:
follow up: Xavier's example

Reply:
I will follow up on this. I think the example might impact performance too much